src/Client/V2/Service/MessageHandler.cs

```diff
@@ -137,6 +137,8 @@ protected override async Task<HttpResponseMessage> SendAsync( HttpRequestMessage
             AddCustomHeaders( request.Headers );
 
             CallContext ctx = RetrieveCallContext( request );
+            if( ctx == null && m_serviceConfig.RequireCallContext )
+                throw new EmsApiNoCallContextException();
```

Comment:
Very minor but I would prefer a blank line after the no-brace if statement when the following line is unrelated

Reply:
Good stuff; fixing.